lib/diagnostic.js

```diff
@@ -47,7 +47,8 @@ let scopes = [/* your response here */];
 // executing it.
 //
 let fireOnAlderaan = function () {
-  // your code here
+  let dSF = deathStar.fire();
+  return dSF.bind(deathStar, alderaan);
```

Comment:
If you want to use bind, you have to call the function: 
```js
let fireOnAlderaan = function () {
  return deathStar.fire.bind(alderaan)()
 };
```

I think the correct answer here is to use `.call()`
```js
let fireOnAlderaan = function () {
  return deathStar.fire.call(alderaan)
 };
```